docs/tutorials/index.rst

```diff
@@ -3,10 +3,9 @@
 =================
 Tutorials
 =================
-The following contain tutorials for how to utilize regolith commands that require editing the regolith.rc file
+The following contain tutorials for how to utilize regolith commands that require a series of steps to perform
 
 .. toctree::
     :maxdepth: 1
 
-    broker
-    mongo_maintenance
\ No newline at end of file
+    broker
```

Comment:
actually, do we want to move the broker instructions to the maintainance tutorials place?  Is the target audience users or maintainers (or developers)?   We could make the toc title less specific, so "Maintainer Docs"  "Developer Docs", something like that?

Reply:
I would say that, even though the broker is a fairly involved tool, the intended audience are still typical users. Maintainers aren't going to have access to everyone's files such that they can create tokens etc. This is all up to the users. 

I'm thinking that database maintenance will ideally envelop how you setup mongo and how you set up your github fs infrastructure. Some broker info can go in the maintenance section, but just the info regarding how you set up the github storage location and distribute access. And I haven't written up anything about that.